lms/templates/instructor/edx_ace/allowedunenroll/push/body.txt

```diff
@@ -0,0 +1,5 @@
+{% load i18n %}
+{% autoescape off %}
+{% blocktrans %}Dear Student,{% endblocktrans %}
```

Comment:
Formality isn't ideal in mobile in my opinion.

```suggestion
```

Reply:
Removed.